### ISSUE TITLE
False alarm to excluded vendor third-party package

### ISSUE THREAD

**Issue by lnshi** [golangci/golangci-lint#476]
`golangci-lint` insists to report:
```
INFO [runner] worker.2 took 281.104182ms with stages: goimports: 281.065566ms
daemon/grpcrp/grpc_reverse_proxy.go:11: File is not `goimports`-ed (goimports)
	"google.golang.org/grpc/codes"
```

Firstly, all vendored projects/packages are excluded, why it insists to report this package?

Secondly, then i specially went to this package, and run `goimports -w`, then it produced NO difference!

**Comment by lnshi** [comment]
@jirfag 

I am not sure i understand u correctly.

`daemon/grpcrp/grpc_reverse_proxy.go` is the source file, `"google.golang.org/grpc/codes"` is one of its imports(and its line number is 11)

If i specially run the `goimports -w daemon/grpcrp/grpc_reverse_proxy.go`, nothing is changed, that mean no issues, right?

Also previously with `gometalinter` never reported this problem.

I updated the goimports: `go get -u golang.org/x/tools/cmd/goimports`, still same: specially run `gomimports` against that file no issue, with `golangci-lint` reports that issue.

My go version: `go version go1.12.1 darwin/amd64`

**Comment by lnshi** [comment]
@jirfag 
I figured out this one, run the `goimports -w daemon/grpcrp/grpc_reverse_proxy.go` in a new terminal, produced some differences:

after the line 11 is:
```
	// Lets register the 'gzip' compressor.
	_ "google.golang.org/grpc/encoding/gzip"
```

so i need to have one blank line before the comments line.
